registry.go

```diff
@@ -80,3 +83,31 @@ func (svc *RegistryServiceOp) Delete(ctx context.Context) (*Response, error) {
 	}
 	return resp, nil
 }
+
+// DockerConfig is the content of a Docker config file
+// that is used by the docker CLI
+// See: https://docs.docker.com/engine/reference/commandline/cli/#configjson-properties
+type DockerConfig struct {
```

Comment:
This is a nit but could you call this struct `DockerCredentials` instead of `DockerConfig`?

Reply:
Sure I can rename it. I went with DockerConfig because what's returned is a config containing the credentials, not only the credentials.